tools/bio3d/visualize_pc.xml

```diff
@@ -0,0 +1,159 @@
+<tool id="bio3d_pc_vis" name="PC visualization" version="@VERSION@">
+    <description>Visualize principle components of motions</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>   
+    <expand macro="requirements" />
+    <command detect_errors="exit_code">
+<![CDATA[ 
+    bio3d_visualize_pc.R 
+        $dcdin 
+        $pdbin
+        $method
+        #if $pca.sele == "calpha":
+        "string"
+        $pca.calpha
+        #end if
+        #if $pca.sele == "cbeta":
+        "string"
+        $pca.cbeta
+        #end if
+        #if $pca.sele == "backbone":
+        "string"
+        $pca.backbone
+        #end if
+        #if $pca.sele == "sidechain":
+        "string"
+        $pca.sidechain
+        #end if
+        #if $pca.sele == "protein":
+        "string"
+        $pca.protein
+        #end if
+        #if $pca.sele == "ligand":
+        "string"
+        $pca.ligand
+        #end if
+        #if $pca.sele == "nucleic":
+        "string"
+        $pca.nucleic
+        #end if
+        #if $pca.sele == "elety":
+        "elety"
+        $pca.elety
+        #end if
+        #if $pca.sele == "resid":
+        "resid"
+        $pca.resid
+        #end if
+        #if $pca.sele == "segid":
+        "segid"
+        $pca.segid
+        #end if  
+        $pdb1  
+        $pc1 
+        $pdb2 
+        $pc2
+        $pdb3
+        $pc3
+    2>&1
+]]></command>
+    <inputs>
+        <param format="dcd" name="dcdin" type="data" label="dcd trajectory input"/>
+        <param format="pdb" name="pdbin" type="data" label="pdb input"/>
+        <param name="method" type="boolean" truevalue="TRUE" falsevalue="FALSE" checked="false" label="Use singular value decomposition (SVD) instead of default eigenvalue decomposition ?" help="Default: No" />
+        <conditional name="pca">
+           <param name="sele" type="select" label="Select domains">
+             <option value="calpha">Calpha</option>
+             <option value="cbeta">Cbeta</option>
+             <option value="backbone">Backbone</option>
+             <option value="sidechain">Sidechain</option>
+             <option value="protein">Protein</option>
+             <option value="ligand">Ligand</option>
+             <option value="nucleic">Nucleic Acids</option>
+             <option value="elety">Atom Names</option>
+             <option value="resid">Resid</option>
+             <option value="segid">Segid</option>
+           </param>
```

Comment:
why not having a select and one simple text input field, without the conditional?